.gitignore

```diff
@@ -1,2 +1,194 @@
 target/
```

Comment:
Instead of specifying every file here, I think you can just specify the 4 - 5 different filenames that should be ignored. I believe they will be ignored from all directories then.